docs/operator-manual/airgap/01-install.md

```diff
@@ -0,0 +1,134 @@
+3. Save policies into a tar.gz file:
+```
+./kubewarden-save-policies.sh --policies-list policies.txt
+```
+kwctl downloads all the policies and stores then in `kubewarden-policies.tar.gz`
+
+## Helm charts
+
+You need to download the following helm charts in your workstation:
+
+```
+helm pull kubewarden/kubewarden-crds         
+helm pull kubewarden/kubewarden-controller
+helm pull kubewarden/kubewarden-defaults  
+helm pull jetstack/cert-manager         
+```
+
+Then move the helm chart tar.gz files to your air-gapped environment.
+
+## Populate private registry
+
+Move `kubewarden-policies.tar.gz`, `kubewarden-images.tar.gz`, `kubewarden-load-images.sh`, `kubewarden-load-policies.sh` and `policies.txt`
+to the air-gapped environment.
+
+1. load Kubewarden images into the private registry
+```
+./kubewarden-load-images.sh --image-list ./kubewarden-images.txt --images kubewarden-images.tar.gz --registry <REGISTRY.YOURDOMAIN.COM:PORT>
+```
+2. load Kubewarden policies into the private registry  
+```
+./kubewarden-load-policies.sh --policies kubewarden-policies.tar.gz --policies-list policies.txt --registry <REGISTRY.YOURDOMAIN.COM:PORT> --sources-path sources.yml 
+```
+
+:::note
+Go to [kwctl documentation](https://docs.kubewarden.io/distributing-policies/custom-certificate-authorities) to learn more about how to configure your sources.yaml
+::: 
+
+## Install kubewarden. 
+
+Let's install Kubewarden now that we have everything we need in our private registry. The only difference with a normal
+Kubewarden installation is that we need to change the registry in the container images and policies to our private registry.
+
+Install cert-manager:
+
+```
+helm install --create-namespace cert-manager ./cert-manager-v1.7.1.tgz \
+    -n kubewarden \
+    --set installCRDs=true \
+    --set image.repository=<REGISTRY.YOURDOMAIN.COM:PORT>/quay.io/jetstack/cert-manager-controller \
+    --set webhook.image.repository=<REGISTRY.YOURDOMAIN.COM:PORT>/quay.io/jetstack/cert-manager-webhook \
+    --set cainjector.image.repository=<REGISTRY.YOURDOMAIN.COM:PORT>/quay.io/jetstack/cert-manager-cainjector \
+    --set startupapicheck.image.repository=<REGISTRY.YOURDOMAIN.COM:PORT>/quay.io/jetstack/cert-manager-ctl
+```
+
+Let's install the Kubewarden stack:
+
+```
+helm install --wait -n kubewarden kubewarden-crds.tgz
+```
+
+```
+helm install --wait -n kubewarden kubewarden-controller.tgz --set common.cattle.systemDefaultRegistry=<REGISTRY.YOURDOMAIN.COM:PORT>
+```
+
+```
+helm install --wait -n kubewarden kubewarden-defaults.tgz --set common.cattle.systemDefaultRegistry=<REGISTRY.YOURDOMAIN.COM:PORT>
+```
+
+Finally, we need to configure PolicyServers to fetch policies from our private registry. See the [using private registry](https://docs.kubewarden.io/operator-manual/policy-servers/private-registry) section of the docs.
+
+Now we can create Kubewarden policies in our cluster! Policies must be available in your private registry.
```

Comment:
BTW, the consultants are fine having the user "figure out" how to do the substitution of the registry